src/component/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import firebase from '../firebase/firebase';
+
+export default class RegisterForm extends Component {
+    state = {
+        firstName: '',
+        lastName: ''
+    }
+
+    changeInput = e => {
+        this.setState({
+            [e.target.name]: e.target.value
+        });
+    }
+
+    addUser = e => {
+        e.preventDefault();
+        const db = firebase.firestore();
+        db.settings({
+            timestampsInSnapshots: true
+        });
+        const userRef = db.collection("users").add({
```

Comment:
@ThorbenBender Shouldn't it just be _db.collection("users).add({ ...._ without the _const userRef = ..._. Because you are not using the userRef anywhere.

Reply:
Refactored this to a promise using then and catch callbacks.